backend/internal/controllers/router.go

```diff
@@ -41,5 +45,8 @@ func GetRouter() chi.Router {
 		r.Get("/", bookController.GetBookList)
 	})
 
+	authenticator := auth.MockAuthenticator{}
```

Comment:
Let's make GetRouter() take an authenticator as a parameter, and let's define this mock authenticator in main.go, with a big // TODO that we need to set up the Firebase project :)